app/src/main/java/com/berniesanders/connect/MainActivity.java

```diff
@@ -32,8 +48,21 @@ protected void onCreate(Bundle savedInstanceState) {
         mText.setText(mContext.getPackageName());
 
         mActionAlertsModel.getActionAlerts().subscribe(
```

Comment:
this leaks an implicit reference to the activity. you should go ahead and leave this as is for now, however...

i can/we should take a quick pass at implementing a subscription manager that wraps composite subscription and uses a conditional binding predicated upon the activity lifecycle.

this is roughly the correct pattern: https://gist.github.com/f2prateek/56e72cca92ad21ba0c38